<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hidato6x6</class>
 <widget class="QMainWindow" name="Hidato6x6">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>727</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>612</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="5">
      <widget class="QTextEdit" name="text_number_23">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QTextEdit" name="text_number_26">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QTextEdit" name="text_number_20">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QTextEdit" name="text_number_22">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QTextEdit" name="text_number_28">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QTextEdit" name="text_number_27">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QTextEdit" name="text_number_25">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QTextEdit" name="text_number_24">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QTextEdit" name="text_number_30">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="4" column="5">
      <widget class="QTextEdit" name="text_number_29">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QTextEdit" name="text_number_31">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QTextEdit" name="text_number_32">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="5">
      <widget class="QTextEdit" name="text_number_35">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QTextEdit" name="text_number_5">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QTextEdit" name="text_number_2">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QTextEdit" name="text_number_8">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QTextEdit" name="text_number_9">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QTextEdit" name="text_number_21">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QTextEdit" name="text_number_11">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QTextEdit" name="text_number_10">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QTextEdit" name="text_number_14">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QTextEdit" name="text_number_16">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QTextEdit" name="text_number_12">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="text_number_13">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QTextEdit" name="text_number_15">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QTextEdit" name="text_number_19">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QTextEdit" name="text_number_17">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QTextEdit" name="text_number_18">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QTextEdit" name="text_number_4">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QTextEdit" name="text_number_6">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QTextEdit" name="text_number_3">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QTextEdit" name="text_number_0">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QTextEdit" name="text_number_34">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QTextEdit" name="text_number_1">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="text_number_7">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QTextEdit" name="text_number_33">
       <property name="font">
        <font>
         <pointsize>32</pointsize>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>791</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_new">
       <property name="text">
        <string>New Game</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_check">
       <property name="text">
        <string>Check Solution</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
